source/Grid/Grid.js

```diff
@@ -1104,6 +1122,9 @@ export default class Grid extends PureComponent {
       if (!autoHeight) {
         newState.scrollTop = scrollTop
       }
+      if (!autoWidth) {
+        newState.scrollLeft = scrollLeft
```

Comment:
Same type of comment here. `scrollLeft` has already been set above, inline, within `newState`.